plugins/modules/archive.py

```diff
@@ -282,6 +286,7 @@ def __init__(self, module):
         self.format = module.params['format']
         self.must_archive = module.params['force_archive']
         self.remove = module.params['remove']
+        self.reproducible_tar = module.params["reproducible_tar"] or False
```

Comment:
Since you provided a default it's either `True` or `False`.
```suggestion
        self.reproducible_tar = module.params["reproducible_tar"]
```